crates/re_ui/src/list_item.rs

```diff
@@ -77,6 +90,26 @@ impl<'a> ListItem<'a> {
         self
     }
 
+    /// Set the subdued state of the item.
+    // TODO(ab): this is a hack to implement the behavior of the blueprint tree UI, where active
+    // widget are displayed in a subdued state (container, hidden space views/entities). One
+    // slightly more correct way would be to override the color using a (color, index) pair
+    // related to the design system table.
+    pub fn subdued(mut self, subdued: bool) -> Self {
+        self.subdued = subdued;
+        self
+    }
+
+    /// Override the hovered state even if the item is not actually hovered.
+    ///
+    /// Used to highlight items representing things that are hovered elsewhere in the UI. Note that
+    /// the [`egui::Response`] returned by [`Self::show`] and ]`Self::show_collapsing`] will still
+    /// reflect the actual hover state.
+    pub fn override_hover(mut self, override_hover: bool) -> Self {
+        self.override_hover = override_hover;
```

Comment:
It's not really an override - setting this to `false` doesn't disable the hovering.

Can we come up with a better name?

Reply:
Right, I guess `force_hovered` is a better named.